File: UNIDAD 1/EXAMEN_EU3_GR10/VIAJECITOS_REST_DOTNET_GR10/02. CLIWEB/VIAJECITOS_REST_CLIWEB_GR10/VIAJECITOS_REST_CLIWEB_GR10/Models/ClienteFacturas.cs
namespace VIAJECITOS_REST_CLIWEB_GR10.Models
{
    public class ClienteFacturas
    {
        public int ClienteId { get; set; }
        public string Nombre { get; set; }
        public string Email { get; set; }
        public string DocumentoIdentidad { get; set; }
        public List<Factura> Facturas { get; set; }
    }
}

File: UNIDAD 1/00_SOA_SIN_BDD/00. SOAP_JAVA_CONUNI_GR10/02. CLIMOV/app/src/main/java/ec/edu/monster/modelo/ClienteFacturas.kt
package ec.edu.monster.modelo

import java.io.Serializable

data class ClienteFacturas(
    var clienteId: Int = 0,
    var nombre: String = "",
    var email: String = "",
    var documentoIdentidad: String = "",
    var facturas: List<Factura> = emptyList()
) : Serializable